contracts/portals/IENSPortal.sol

```diff
@@ -0,0 +1,26 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+pragma solidity ^0.8.8;
+
+import {IPortal} from "./IPortal.sol";
+
+/// @title ENS Portal interface
+interface IENSPortal is IPortal {
+    // Errors
+
+    /// @notice ENS input can only be sent by its owner
+    error ENSOnlyOwner();
+
+    // Permissionless functions
+
+    /// @notice Send ENS input to InputBox
+    /// @param node The ENS node
+    /// @param appContract The application contract address
+    /// @param execLayerData Additional data to be interpreted by the execution layer
+    function sendENS(
```

Comment:
Again, it's more than just the ENS node. You're telling the machine that the ENS node resolves to the Ethereum address of whoever sent that input.
Maybe we could rename it to something like `sendInputWithENSNode`?